src/lib/services/user-service.ts

```diff
@@ -297,6 +309,7 @@ class UserService {
         });
         if (allowed) {
             await this.changePassword(user.id, password);
+            await this.sessionService.deleteSessionsForUser(user.id);
```

Comment:
Nice. This will work regardless of where the reset-password is triggered from (admin user, API, the user itself) :+1:

Reply:
And should also only delete sessions if we actually manage to change the password.